src/utils/getPackageVersion.ts

```diff
@@ -0,0 +1,17 @@
+import { promisify } from 'util';
+import { exec } from 'child_process';
+
+const execPromise = promisify(exec);
+
+export const getPackageVersion = async (name: string): Promise<string> => {
+  const { stdout: output, stderr: error } = await execPromise(
+    `npm view ${name} version`,
+  );
+  const versionInfo = output.toString().trim();
+
+  if (error) {
+    throw new Error(error);
+  }
+
+  return versionInfo;
```

Comment:
This function returns the package version but you only seem to care if a package exists since the result of this function doesn't get used. Can u elaborate on this on why u think it's necessary to return this version info here instead of only throwing a error|null?

Reply:
Yes, you're right. Before I thought to use this information, but it turns out, the error handling is enough for now. It's removed and also the function name has been changed to something more descriptive.